chef/cookbooks/crowbar/templates/default/crowbar-delete-pacemaker-resources.sh.erb

```diff
@@ -29,6 +29,13 @@ rm -f $UPGRADEDIR/crowbar-delete-pacemaker-resources-failed
 
 <% if @use_ha %>
 
+log "Resetting rabbitmq cluster..."
+rabbitmq_nodes=$(rabbitmqctl cluster_status | tr -d '\n' | tr -d ' ' | sed -r -e 's/.*\{disc,\[//' -e 's/\].*//' -e "s/'//g" -e 's/,/ /g')
```

Comment:
Do we have to ask `rabbitmqctl cluster_status` for the nodes? Can't we generate them based on the chef roles?

Reply:
well... it's a trust issue :)

Reply:
Note that what cluster_status returns is exactly the input we need for later commands. If we generate it based on roles, there will be two places with no clear connection, one naming the nodes and another one assuming some naming scheme.